### PR TITLE
Homepage\Dashboard

### LINKED ISSUE #135: Create dashboard
Logged in users should have a dashboard where they can have an overview of the features of the platform.
It should act as a redundant navigation system in case the circular menu isn't obvious.
This also solves the problem of what page the user lands on after login or registration.

### PR DESCRIPTION
**Pull request by LordOfTheNeverThere**
In this Pull Request we Designed the Dashboard or Homepage, which will be shown to users if they are logged in. On the Root address. It has some basic navigation, and future aspects we will address as we build the site, communicate to the users. Due note that this is quite primitive, this page will need improvement, a more dynamic and robust a approach of showing information is perhaps beneficial, and responsiveness (Scroll bars, cascade elements...)

Closes #135

### CODE REVIEW

File: frontend/src/pages/HomePage.js [hunk 1]
```diff
@@ -103,6 +104,20 @@ export const HomePage = () => {
...
+						<Grid item xs={3}>
+							<IconButton
+								direction='column'
+								fontSize={25}
+								text='Contribui com perguntas'
+								iconFirst={false}
+								alignItems='flex-start'
+								height='23vh'
+								width='12vw'
+								href='/leaderboards'
```
Comment (JeronimoMendes) [inline frontend/src/pages/HomePage.js:116]: Fix the href

File: frontend/src/pages/HomePage.js [hunk 2]
```diff
@@ -1,22 +1,166 @@
...
+							<IconButton
+								direction='column'
+								fontSize={25}
+								text='Ver o teu perfil'
+								iconFirst={false}
+								alignItems='flex-start'
+								height='23vh'
+								width='12vw'
+								href='/perfil'
+							>
```
Comment (JeronimoMendes) [inline frontend/src/pages/HomePage.js:89]: image
This icon is misaligned with the rest

**Review by JeronimoMendes — CHANGES_REQUESTED** [review]

File: frontend/src/pages/HomePage.js (hunk 1 above)
Comment (LordOfTheNeverThere) [reply frontend/src/pages/HomePage.js]: You're right! Thanks!

File: frontend/src/pages/HomePage.js (hunk 2 above)
Comment (LordOfTheNeverThere) [reply frontend/src/pages/HomePage.js:89]: Since it's a grid the distance will depend on the overflow of the text. But I will tr to find a solution

Comment (LordOfTheNeverThere) [reply frontend/src/pages/HomePage.js]: @JeronimoMendes Check It out Now
image

**Review by JeronimoMendes — APPROVED** [review]

**Pull request merged**